### PR TITLE
Jira keys

### PR DESCRIPTION
**Pull request by xendk**
A lot of cleanup, just to be able to write `PROJ-123` instead of `#PROJ-123`.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,15 +14,15 @@
...
+    "php": ">=7.2.0",
     "gridonic/hapi": "dev-master",
-    "symfony/console": "^3.4",
-    "symfony/yaml": "^3.4",
-    "alexkovalevych/jira-api": "^1.0.6",
-    "mnapoli/silly-php-di": "^1.2"
+    "symfony/console": ">=3.4",
+    "symfony/yaml": ">=3.4",
+    "alexkovalevych/jira-api": ">=1.0.6",
+    "mnapoli/silly-php-di": ">=1.2"
```
Comment (kasperg) [inline composer.json:22]: Do we really want to go with *anything* newer than the specified versions? Other libraries usually go with something like `"symfony/console": "^3.4||^4||^5"`.

File: src/BugTracker/Jira.php [hunk 2]
```diff
@@ -259,13 +248,4 @@ private function formatComment($timelog)
             )
         );
     }
-
-    /**
-     * Preparing this for JIRA, e.g. removing #hashmark, transforming "#scl-123" to "SCL-123"
-     */
-    public function sanitizeTicketId($ticketId)
-    {
-        $ticketId = trim(strtoupper(str_replace("#", "", $ticketId)));
```
Comment (kasperg) [inline src/BugTracker/Jira.php:268]: By removing this function entirely it also seem as if we drop the call to `strtoupper()`. Will that cause problems?

**Review by kasperg — APPROVED** [review]
:+1: A few questions but otherwise LGTM 😵 .

File: composer.json (hunk 1 above)
Comment (xendk) [reply composer.json:22]: In order to support PHP 7.1 to 7.4, yes...

But I've later dropped 7.1, so maybe not.

File: src/BugTracker/Jira.php (hunk 2 above)
Comment (xendk) [reply src/BugTracker/Jira.php:268]: `extractIds()` already uppers the ticket IDs. As far as I could tell, the only place it was used, it was used on a string gotten from `extractIds()`.

File: composer.json (hunk 1 above)
Comment (xendk) [reply composer.json]: I've fixed the dependencies to something more conventional.

**Pull request merged**